### ISSUE TITLE
Inquiry about DETR result

### ISSUE THREAD

**Issue by JCTaylor666** [CAMMA-public/SurgLatentGraph#10]
Hi, thank you for sharing your work!

I noticed that your reported mAP for object detection using Deformable DETR is 34.8. I'm curious whether you've also tried using vanilla DETR (i.e., the original DETR without deformable attention) in your experiments.

If so, could you please share:

The mAP you achieved with vanilla DETR

The training settings you used (e.g., number of epochs, learning rate, batch size, etc.)

Thanks in advance for your help!

**Comment by adit98** [comment]
Hi,

Unfortunately we haven't tried with Vanilla DETR - but it should be straightforward to try, especially just for the object detection part - you can just grab the corresponding config file from the mmdetection repo and modify the dataset.
